website/src/release.md

```diff
@@ -202,6 +202,35 @@ please cancel the release for the current `release_version`,
 _increase th RC counting_ and re-initiate a release with the new `release_version`.
 And remember to delete the wrong artifacts from the SVN dist repo.
 
+## How to verify a release
```

Comment:
nit: move this after the "Voting" section below, which sends out the email to vote for RC